perun-core/src/main/java/cz/metacentrum/perun/core/bl/MembersManagerBl.java

```diff
@@ -638,6 +610,24 @@
 	List<RichMember> getCompleteRichMembers(PerunSession sess, Group group, List<String> attrsNames, List<String> allowedStatuses, boolean lookingInParentGroup) throws InternalErrorException, AttributeNotExistsException, ParentGroupNotExistsException;
 
 	/**
+	 * Get all RichMembers with attributes specific for list of attrNames.
+	 * Attributes are defined by member (user), group, resource (facility) objects.
+	 * Members are defined by group and are filtered by list of allowed statuses.
+	 *
+	 * @param sess
+	 * @param group
+	 * @param resource
+	 * @param attrsNames
+	 * @param allowedStatuses
+	 * @return list of richMembers with specific attributes
+	 * @throws InternalErrorException
+	 * @throws AttributeNotExistsException
+	 * @throws ParentGroupNotExistsException
```

Comment:
``ParentGroupNotExistsException`` can be removed. It was a check for case, when admin searched parent group for listing members to add to group, but you don't use it in this method, neither we call it from GUI in original copy/pasted method.